src/main/java/core/basesyntax/shape/Shape.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.behaviour.Colorable;
+import core.basesyntax.behaviour.Drawable;
+import core.basesyntax.behaviour.Geometrical;
+import core.basesyntax.type.Color;
+
+public abstract class Shape
+        implements Colorable, Geometrical, Drawable {
+    public static final byte SHAPES_COUNT = 5;
+
+    private Color color;
```

Comment:
make field protected or add relevant getter to make property accessible from child classes